pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -91,6 +95,27 @@ const DepositDates = ({
         )}
     </Card>
 
+    <Card id="crossRepositoryCheck" tag="section">
+      <Card.Title tag="h2">{texts.crossRepositoryCheck.title}</Card.Title>
+      <Card.Description>
+        {texts.crossRepositoryCheck.description}
+      </Card.Description>
+      <p>
+        {store.depositDates.crossDepositLag
+          ? texts.crossRepositoryCheck.body.render({
+              nonCompliantCount:
+                store.depositDates.crossDepositLag.nonCompliantCount,
+              recordsInAnotherRepository:
+                store.depositDates.crossDepositLag.possibleBonusCount,
+              days: '#', // TODO: Where to get this value?
```

Comment:
Maybe we should not have this before we have actual table data.

Reply:
You mean the whole card or only the last few words?

Reply:
I mean exactly `days`.